src/Libraries/CoreNodeModels/Properties/Resources.en-US.resx

```diff
@@ -584,4 +584,22 @@
   <data name="SelectionPointsOutputPortName" xml:space="preserve">
     <value>Points</value>
   </data>
+  <data name="ListGroupByKeyDescription" xml:space="preserve">
+    <value>Group items into sub-lists based on their like key values</value>
+  </data>
+  <data name="ListGroupByKeyGroupsTooltip" xml:space="preserve">
+    <value>list of sublists, with items grouped by like key values</value>
+  </data>
+  <data name="ListGroupByKeyKeysTooltip" xml:space="preserve">
+    <value>Key values, one per item in the input list, used for grouping the items</value>
```

Comment:
a though / question for @mccrone should we mention issues that can arise when using complex objects or doubles as keys?